docs/config/shared-options.md

```diff
@@ -163,6 +163,13 @@ Enabling this setting causes vite to determine file identity by the original fil
 - **Related:** [esbuild#preserve-symlinks](https://esbuild.github.io/api/#preserve-symlinks), [webpack#resolve.symlinks
   ](https://webpack.js.org/configuration/resolve/#resolvesymlinks)
 
+## html.cspNonce
+
+- **Type:** `string`
+- **Related:** [Content Security Policy (CSP)](/guide/features#content-security-policy-csp)
+
+A nonce value placeholder that will be used when generating script / style tags. Setting this value will also generate a meta tag with nonce value.
+
```

Comment:
One last idea, sorry for the late comment here. Should we make this one:
`{ csp: { nonce: ... } }` in case we later on have other csp related features we want to add here and maybe top level as it may affect other things later on (headers?).

Reply:
SvelteKit has its own csp config: https://kit.svelte.dev/docs/configuration#csp. I think it's unlikely it will support ours though since it has more features controlling the entire app. I think `html.cspNonce` makes sense to me for now too.

Maybe later we can document that the `html` option and `transformIndexHtml` is Vite-specific and may not work in other frameworks.

Reply:
Ok, let's go with `html.cspNonce` then. Thanks for the discussion!